packages/appstore/src/components/main-title-bar/__tests__/main-title-bar.spec.tsx

```diff
@@ -0,0 +1,206 @@
+import React from 'react';
```

Comment:
@aizad-deriv A suggestion for reordering the imports based on this [document](https://github.com/binary-com/deriv-app/blob/master/docs/JavaScript/README.md#import-rules) 🙏